dev-client/src/translations/en.json

```diff
@@ -513,6 +524,14 @@
       "clay_label": "Percent clay",
       "clay_help": "0–100%",
       "fragment_title": "Rock Fragment Volume",
+      "fragment": {
+        "info": {
+          "title": "Rock Fragment Volume",
+          "description1": "Rock fragment volume is a measurement of how much of the soil volume is taken up by rocks, which is defined as anything larger than 2mm (1/16”) in diameter.",
```

Comment:
issue: separate this out so we either show `2mm` or `1/16"`, depending on the user's unit preference. See `vertical_cracking.description`.

Reply:
Ah, will do. 
Out of curiosity -- it looks like the components are hard-coding `units: 'METRIC'`. Are there plans for metric vs. english to be a user setting for a particular release target?

Reply:
Definitely separate that other string, too. Thanks for catching it.

A settings for units will be enable prior to release.

Reply:
Will add to refactoring list to not clutter this PR